js_modules/dagster-ui/packages/ui-core/src/graph/KindTags.tsx

```diff
@@ -13,13 +13,16 @@ export const STORAGE_KIND_TAG = 'dagster/storage_kind';
 export const KIND_TAG_PREFIX = `dagster/kind/`;
 
 // Older code servers may be using the legacy compute kind tag, so we need to check for both
-export const isCanonicalComputeKindTag = (tag: DefinitionTag) =>
+export const isCanonicalComputeKindTag = (tag: Omit<DefinitionTag, '__typename'>) =>
```

Comment:
This seems good but these could just be `{key: string}` which might be more correct becasue they're not always definition tags? (Or maybe they are all called that now)